### ISSUE TITLE
False positive on Style/VariableNumber

### ISSUE THREAD

**Issue by jaredmoody** [bbatsov/rubocop#3699]
I don't think the variable `@__app__` should trigger this cop, but it does.

```
> rubocop -V                                                                                                                                  
0.45.0 (using Parser 2.3.1.4, running on jruby 2.3.1 java)
```

**Comment by bquorning** [comment]
This indeed looks like a bug.

I’m working on a fix, will open a PR shortly.
